yarn-error.log

```diff
@@ -0,0 +1,2304 @@
+Arguments: 
```

Comment:
I think that log should not be in commit :stuck_out_tongue:

Reply:
![fizzer1k-5597](https://user-images.githubusercontent.com/6560742/206736962-3284f86e-4479-425e-b723-6bfef6b99758.gif)

Reply:
Might as well add it to the gitignore file, while I'm at it.